GameObjectExtensions.cs

```diff
@@ -20,22 +34,47 @@ public static TBehaviour CreateChild<TBehaviour>(this GameObject parent)
 			return behaviour;
 		}
 
-		public static TBehaviour CreateChild<TBehaviour>(this GameObject parent, string path, bool worldPositionStay = true)
+		/// <summary>
+		/// Using the given path to load and create the asset as a child transform that has the given TBehaviour component.
```

Comment:
Try to rephrase this more similar to the others.

`Creates a new child object, by loading from resources and instantiating`

You probably don't need to mention the path or the given `TBehaviour` since you will mention it in the `param` tags